bin/test-workflow/2_admin_load_conjur_policies.sh

```diff
@@ -7,13 +7,15 @@ PLATFORM="${PLATFORM:-kubernetes}"
 
 source utils.sh
 
+export CONJUR_ADMIN_PASSWORD="$(get_admin_password)"
```

Comment:
Hmmm... this would be much cleaner if we modified the Conjur OSS Helm chart to accept an admin password, so that this test setup would be the same for Conjur OSS vs. Enterprise.

Your call on this, but I'm wondering if it would be a little bit clearer, but a tad chattier, to only call the `get_admin_password` function for Conjur OSS?